cms/static/cms/js/modules/cms.structureboard.js

```diff
@@ -295,6 +295,8 @@ $(document).ready(function () {
 			this.interval = setInterval(function () {
 				$(window).trigger('resize.sideframe');
 			}, interval);
+
+			$(window).trigger('structureboard_shown.sideframe');
```

Comment:
@FinalAngel Do we have an inventory anywhere of all the custom events we trigger/publish anywhere?

@qris raises a good point here.